DragAndDrop/View.swift

```diff
@@ -0,0 +1,72 @@
+//
+//  View.swift
+//  DragAndDrop
+//
+//  Created by Stanley Chiang on 3/6/16.
+//  Copyright © 2016 SantaClaraiOSConnect. All rights reserved.
+//
+
+import UIKit
+
+protocol ViewDelegate {
+    func reset()
+}
+
+class View: UIView, draggableViewDelegate {
+
+    var draggableView:DraggableView!
+    var trashArea:UIView!
+    var resetButton:UIButton!
+    
+    var delegate:ViewDelegate!
+    
+    override init(frame: CGRect) {
+        super.init(frame: frame)
+        
+        trashArea = initTrashArea()
+        addSubview(trashArea)
+        
+        resetButton = initResetButton()
+        addSubview(resetButton)
+        
+        draggableView = initDraggableView()
+        addSubview(draggableView)
+        
+    }
```

Comment:
This comment is on the initializer. 

We don't need it. Since we are already init-ing our views above. 
We should add our subViews by 
```swift
override layoutSubviews() {
    addSubView(trashArea)
    ...
}
```

Once we have done that we don't need this initializer any more, 
And for the super win we don't need that stupid `required init?(coder aDecoder: NSCoder)` one either :100: